build/releasing/index.ts

```diff
@@ -6,11 +6,9 @@ export const config: BeachballConfig = {
     groups: [
         {
             name: "Microsoft FAST",
-            include: [
-                "packages/tooling/*",
-                "packages/utilities/*",
-                "packages/web-components/*",
-            ],
+            include: ["packages/utilities/*", "packages/web-components/*"],
```

Comment:
I think you want to delete the first `include` field

Reply:
I guess the web merge doesn't work so well.  Resolved.